api/capi/include/ml-api-common.h

```diff
@@ -18,11 +19,14 @@
 #ifndef __ML_API_COMMON_H__
 #define __ML_API_COMMON_H__
 
-#include <tizen_error.h>
-
 #ifdef __cplusplus
 extern "C" {
 #endif /* __cplusplus */
+
+#ifdef __TIZEN__
```

Comment:
If this file is shipped on Tizen package, `#ifdef __TIZEN__` is not allowed.

Reply:
Thank you for the confirmation, I have made a change, as per request.

as `ml-api-common.h` is a part of `ml-api-common` package. It should be available across platforms. So, I renamed it to be `ml-api-common.h`
 Could you take another look?